docs/ctapipe_api/calib/index_camera.rst

```diff
@@ -9,20 +9,19 @@ Camera calibration
 Introduction
 ============
 
-This directory contains all the functions that are used to calibrate the CTA
-Cameras (MC, prototypes and final camera calibration algorithms).
-
-The calibration is seperated between data level transitions:
-    * R0 -> R1 (Camera precalibration) (r1.py)
-    * R1 -> DL0 (Data volume reduction) (dl0.py)
-    * DL0 -> DL1 (Waveform cleaning and charge extraction) (dl1.py)
-
-The routines in these scripts take a ctapipe event container, and fill the
-next data level container with the calibrated information.
-If the container is empty for the source data level then that calibration is
-skipped and the target data level container is unchanged (for example
-performing the calibration in r1.py on data read into the R1 container will
-leave the container unchanged, as there is no data in the R0 container)
+This module contains all the methods and classes that are used to calibrate the
+CTA Cameras (MC, prototypes and final camera calibration algorithms).
+
+CameraCalibrator
+==========
```

Comment:
I think this underline is what is stopping the doc build (just needs to be the right length).  Also, it's at the same heading level (`===`) as the title "Charge Extraction".  If it's supposed to be 1-level lower, use `---` (see here https://documentation-style-guide-sphinx.readthedocs.io/en/latest/style-guide.html)